### PR TITLE
patch(ManagerSideNav): Add type

### PR DESCRIPTION
**Pull request by circAssimilate**
**Summary:**
Per [this Slack thread](https://optimizely.slack.com/archives/CNPTR622U/p1589908899084200), we'll need to export types for components from https://github.com/optimizely/oui/commit/401506bd583befd1e53e1dbd18312599ac517cef for them to work in repos like Flags that use TypeScript. This runs `yarn generate-types` to remedy that. It also exports `ManagerSideNav` from _types/templates/module-declaration.d.ts_.

**Test Plan:**
Manually testing via `yarn link` before merging.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This file is similar to the format suggested by [Keep a CHANGELOG](https://github.com/olivierlacan/keep-a-changelog).
 
 ## Unreleased
+- [Patch] Export `ManagerSideNav` info for TypeScript ([#1334](https://github.com/optimizely/oui/pull/1334))
```
Comment (ayeshakmaz) [inline CHANGELOG.md:9]: Could you switch to bolding the component name? Generally component names are bold to help with scanning and prop names are monospaced

Comment (circAssimilate) [reply CHANGELOG.md]: For sure - copy / paste fail on my part. Thank you!

**Comment by circAssimilate** [comment]
Roger that on the important ordering. Thanks, @jamesopti!

**Comment by ayeshakmaz** [comment]
@jamesopti How did you know they'd been run out of order?

File: types/index.d.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ declare module 'optimizely-oui' {
   export { default as Label } from 'components/Label/index';
   export { default as Link } from 'components/Link/index';
   export { default as ListGroup } from 'components/ListGroup/index';
+  export { default as ManagerSideNav } from 'components/ManagerSideNav/index';
```
Comment (jamesopti) [inline types/index.d.ts:38]: 👍

File: types/templates/module-declaration.d.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ declare module 'optimizely-oui' {
   export { default as Label } from 'components/Label/index';
   export { default as Link } from 'components/Link/index';
   export { default as ListGroup } from 'components/ListGroup/index';
+  export { default as ManagerSideNav } from 'components/ManagerSideNav/index';
```
Comment (jamesopti) [inline types/templates/module-declaration.d.ts:38]: 👍

File: types/index.d.ts (hunk 2 above)
Comment (jamesopti) [reply types/index.d.ts:38]: Thanks! This is what creates the link from `ManagerSideNav` to the below defined "'components/ManagerSideNav/index'"

**Pull request merged**